terraform/snowflake/modules/policies/main.tf

```diff
@@ -130,16 +151,14 @@ resource "snowflake_legacy_service_user" "sentinel" {
   lifecycle {
     ignore_changes = [rsa_public_key]
   }
-
-  # Use the input variable here
-  default_warehouse = var.logging_warehouse_name
-  # Use the input variable here
-  default_role      = var.logger_role_name
+# no cross-module references; use resources created here
```

Comment:
This comment doesn't seem particularly helpful, as it refers to a review conversation rather than anything in the code